src/AcmCountCard/AcmCountCard.tsx

```diff
@@ -185,6 +186,7 @@ export const AcmCountCard = (props: AcmCountCardProps) => {
             onClick={props.onCardClick}
             isSelectable={!!props.onCardClick}
             isFlat={!props.onCardClick}
+            onKeyPress={props.onKeyPress}
```

Comment:
I think this leaves it open for any key to trigger the event. Not sure if we want to restrict it to only enter & space keys?

Reply:
True - My thought was not limiting that in shared component and allowing that to be defined by end user.  `space` triggers opening of kebab-menu so if user tabs one too many/less they may fire card instead of intended drill down into menu.